scripts/Imaging/IMAT/tomo_reconstruct.py

```diff
@@ -69,99 +69,192 @@ def setup_cmd_options():
-                          "of the slices. Empty or zero implies no masking.")
-
-    grp_post.add_argument("--cut-off", required=False, type=float,
-                          help="Cut off level (percentage) for reconstructed "
-                          "volume. pixels below this percentage with respect to maximum intensity in the stack "
-                          "will be set to the minimum value.")
-
-    grp_post.add_argument("--out-median-filter", required=False, type=float,
-                          help="Apply median filter (2d) on reconstructed volume with the given window size.")
-
-    parser.add_argument("-v", "--verbose", action="count", default=1, help="Verbosity level. Default: 1. "
-                        "User zero to supress outputs.")
+    grp_pre.add_argument(
+        "--in-img-format",
+        required=False,
+        default='fits',
+        type=str,
+        help="Format/file extension expected for the input images. Supported: {0}".
+        format(img_formats))
+
+    grp_pre.add_argument(
+        "--out-img-format",
+        required=False,
+        default='tiff',
+        type=str,
+        help="Format/file extension expected for the input images. Supported: {0}".
+        format(img_formats))
+
+    grp_pre.add_argument(
+        "--region-of-interest",
+        required=False,
+        type=str,
+        help="Region of interest (crop original "
+        "images to these coordinates, given as comma separated values: x1,y1,x2,y2. If not "
+        "given, the whole images are used.")
+
+    grp_pre.add_argument(
+        "--air-region",
+        required=False,
+        type=str,
+        help="Air region /region for normalization. "
+        "If not provided, the normalization against beam intensity fluctuations in this "
+        "region will not be performed")
+
+    grp_pre.add_argument(
+        "--median-filter-size",
+        type=int,
+        required=False,
+        help="Size/width of the median filter (pre-processing")
+
+    grp_pre.add_argument(
+        "--remove-stripes",
+        default='wf',
+        required=False,
+        type=str,
+        help="Methods supported: 'wf' (Wavelet-Fourier)")
+
+    grp_pre.add_argument(
+        "--rotation",
+        required=False,
+        type=int,
+        help="Rotate images by 90 degrees a number of "
+        "times. The rotation is clockwise unless a negative number is given which indicates "
+        "rotation counterclocwise")
+
+    grp_pre.add_argument(
+        "--scale-down",
+        required=False,
+        type=int,
+        help="Scale down factor, to reduce the size of "
+        "the images for faster (lower-resolution) reconstruction. For example a factor of 2 "
+        "reduces 1kx1k images to 512x512 images (combining blocks of 2x2 pixels into a single "
+        "pixel. The output pixels are calculated as the average of the input pixel blocks."
```

Comment:
Does that mean the output resolution is scaled down or is it the same resolution based on the interpolated values?